cmd/juju/status/status_test.go

```diff
@@ -171,8 +171,9 @@ var (
 			"current": "started",
 			"since":   "01 Apr 15 01:23+10:00",
 		},
-		"dns-name":    "controller-0.dns",
-		"instance-id": "controller-0",
+		"dns-name":     "controller-0.dns",
+		"ip-addresses": []string{"controller-0.dns"},
```

Comment:
+1; I'd also suggest adding tests with full IPv6 addresses in the range reserved for documentation purposes (2001:db8::/32).